### ISSUE TITLE
Exception running with various mods on 1.19.2.

### ISSUE THREAD

**Issue by Dreytac** [CyclopsMC/IntegratedREST#23]
#### Issue type:

- :bug: Bug

____

#### Short description:

Upon adding Integrated REST to my modpack, Minecraft crashes as startup with the following exception:

```
Exception in thread "main" java.lang.module.ResolutionException: Modules integratedrest and io.netty.resolver export package io.netty.resolver to module mekanism
	at java.base/java.lang.module.Resolver.resolveFail(Resolver.java:901)
	at java.base/java.lang.module.Resolver.failTwoSuppliers(Resolver.java:815)
	at java.base/java.lang.module.Resolver.checkExportSuppliers(Resolver.java:736)
	at java.base/java.lang.module.Resolver.finish(Resolver.java:380)
	at java.base/java.lang.module.Configuration.<init>(Configuration.java:140)
	at java.base/java.lang.module.Configuration.resolveAndBind(Configuration.java:494)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.ModuleLayerHandler.buildLayer(ModuleLayerHandler.java:75)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:60)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.run(Launcher.java:104)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
```

The modules range from Mekanism to Caelus API to Dismount Entity (those I've observed so far, seems random each time I load). Integrated REST appears to be the common cause.

#### Steps to reproduce the problem:

1. Add Caelus API, Dismount Entity or Mekanism mods along with Integrated REST (and it's requirements).
2. Observe exception on startup.

#### Expected behaviour:

Minecraft should load without crashing.
____

#### Versions:

- This mod: 1.1.6
- Minecraft: 1.19.2
- Forge: 43.1.3
- Mekanism: 10.3.3.470
- Dismount Entity: 1.8
- Caelus API: 3.0.0.6

#### Log file:

https://pastebin.com/AwkxXfym

**Comment by rubensworks** [comment]
Thanks for reporting!

**Comment by rubensworks** [comment]
At first glance, it looks like the jar shadowing is failing somehow.

**Comment by LobaCW** [comment]
Hi, I am playing in a modpack and after I installed this mod the same problem happens, the only difference is that I can load the world for a few minutes before it crashes, but the modules is random each time as well, so I figure is the same problem as this one since it has been fixed for version 1.19.
I was wondering, could you fix this issue for minecraft 1.12? Would that be easy to implement for this version?
I didn't want to open another issue because it would be duplicate of this one.
Thanks in advance.

**Comment by rubensworks** [comment]
I'm sorry, but MC 1.12 is not supported anymore.
